README.md

```diff
@@ -35,7 +35,7 @@ import ChipInput from 'material-ui-chip-input'
 | --- | --- | --- | --- |
 | allowDuplicates | `bool` | `false` | Allows duplicate chips if set to true. |
 | chipContainerStyle | `object` | | Override the inline-styles of the chip container element. |
-| chipRenderer | `function` | | A function of the type `({ value, text, isFocused, isDisabled, handleClick, handleRequestDelete, defaultStyle }, key) => node` that returns a chip based on the given properties. This can be used to customize chip styles.  Each item in the `dataSource` array will be passed to `chipRenderer` as arguments `value` and `text`. If `dataSource` is an array of objects and `dataSourceConfig` is present, then `value` and `text` will instead correspond to the object values defined in `dataSourceConfig`. If `dataSourceConfig` is not set and `dataSource` is an array of objects, then a custom `chipRenderer` must be set.|
+| chipRenderer | `function` | | A function of the type `({ value, text, chip, isFocused, isDisabled, handleClick, handleRequestDelete, defaultStyle }, key) => node` that returns a chip based on the given properties. This can be used to customize chip styles.  Each item in the `dataSource` array will be passed to `chipRenderer` as arguments `value` and `text`. If `dataSource` is an array of objects and `dataSourceConfig` is present, then `value` and `text` will instead correspond to the object values defined in `dataSourceConfig`. If `dataSourceConfig` is not set and `dataSource` is an array of objects, then a custom `chipRenderer` must be set.|
```

Comment:
No problem, I changed it.

> Each item in the `dataSource` array will be passed to `chipRenderer` as arguments `chip`, `value` and `text`.

I think it describes the functionality accurately, but might not be verbose enough.